services/traction/api/endpoints/routes/v1/tenant/contacts/contact.py

```diff
@@ -33,7 +33,7 @@ async def get_contact(
     wallet_id = get_from_context("TENANT_WALLET_ID")
     tenant_id = get_from_context("TENANT_ID")
 
-    item = await contacts_service.get_contact(
+    db_item = await contacts_service.get_contact(
```

Comment:
this service call returns a ContactItem, the transformation in the route isn't needed. we will discuss, but these API services should be returning API level items not db

Reply:
Yea i didn't got back and redo this after talked about it. 

Where do we want to put data operations? in the actual sqlmodel class? or do you see the need for 'Data Access Layer'

Internally we may want to update a contact as a side-effect, so I want to make sure that the 'update-contact' method isn't coupled with the endpoint. That's the only reason i thought we might want to do it this way. 

I'll make it consistent with the rest for now. my b

Reply:
we will probably need a data access service at some point, just didn't want to go there until needed. i figure once we are worrying about cross table transactions we will need it.  i think for now, what you did for the fetch by various ids/fields in the sqlmodel is great. updates/deletes/searches can remain in the existing service layer. if we find we are recoding those exact updates/deletes/searches we can refactor out.